packages/protocol/contracts/layer1/preconf/libs/LibPreconfConstants.sol

```diff
@@ -23,6 +23,16 @@ library LibPreconfConstants {
     uint256 internal constant RANDOMNESS_DELAY_EPOCHS = 2;
 
     bytes32 internal constant PRECONF_DOMAIN_SEPARATOR = keccak256("TAIKO_ALETHIA_PRECONF");
+    bytes32 internal constant ADD_OVERSEER_SIGNER_DOMAIN_SEPARATOR =
+        keccak256("TAIKO_ALETHIA_ADD_OVERSEER_SIGNER");
+    bytes32 internal constant REMOVE_OVERSEER_SIGNER_DOMAIN_SEPARATOR =
+        keccak256("TAIKO_ALETHIA_REMOVE_OVERSEER_SIGNER");
```

Comment:
if a constant is only used in one contract, maybe we should move it to the contract.